charts/rapid-response/templates/clusterrole.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.rbac.create  }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "rapidResponse.fullname" .}}
+  labels:
+{{ include "rapidResponse.labels" . | indent 4 }}
+rules:
+  - apiGroups:
+      - ''
+      - 'rbac.authorization.k8s.io'
+      - 'extensions'
+      - 'apps'
+      - 'batch'
+      - 'networking.k8s.io'
+      - 'autoscaling'
+      - 'policy'
+      - 'storage.k8s.io'
+    resources:
```

Comment:
does it need all these resources?

Reply:
I am not sure at the moment but keeping it consistent with the other charts. If needed an evaluation can be done in some other ticket on what all would be required.

Reply:
@yashgiri @ironashram Actually the rapid response doesn't require those resources, it just need his daemonset+configmaps/secrets
https://docs.sysdig.com/en/docs/installation/rapid-response-installation/#as-kubernetes-daemonset